lib/ex_unit/lib/ex_unit/runner_stats.ex

```diff
@@ -25,6 +25,13 @@ defmodule ExUnit.RunnerStats do
     {:noreply, %{map | total: total + 1, skipped: skipped + 1}}
   end
 
+  def handle_cast({:case_finished, %ExUnit.TestCase{state: {:failed, _reason}} = test_case},
+                  %{failures: failures, total: total} = map) do
+
```

Comment:
Sorry for the nitpick but please remove this empty line. Everything else is perfect! :D Thank you.